src/Microsoft.AspNetCore.SignalR.Core/HubConnectionHandler.cs

```diff
@@ -2,6 +2,8 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+using System.Collections.Generic;
+using System.Linq;
```

Comment:
LINQ....

Reply:
Would you prefer `SupportedProtocols` on `HubOptions` was also a readonly list?

Reply:
It needs to be mutable for users to tweak